app/views/shared/_npc_show.html.erb

```diff
@@ -0,0 +1,121 @@
+  <p>Background: <%= @npc.background_name %></p>
+  <p>Class: <%= @npc.character_class %></p>
+  <p>Alignment: <%= @npc.alignment %></p>
+  <p>Speed: <%= @npc.speed %></p>
+  <p>Initiative: <%= @npc.initiative %></p>
+  <p>Armor Class: <%= @npc.armor_class %></p>
+  <p>Size: <%= @npc.size %></p>
+  <p>Vision: <%= @npc.vision %></p>
+  <p>Hit Dice: <%= @npc.hit_dice %></p>
+  <p>Languages: <%= @npc.languages %></p>
+
+  <h3>Personality:</h3>
+    <p>Personality: <%= @npc.background_personality_personality %></p>
+    <p>Ideal: <%= @npc.background_personality_ideal %></p>
+    <p>Bond: <%= @npc.background_personality_bond %></p>
+    <p>Flaw: <%= @npc.background_personality_flaw %></p>
+
+  <h3>Equipment:</h3>
+    <p>Gold: <%= @npc.equipment_gold %></p>
+    <p>Pack: <%= @pack[0].name %></p>
+    <p>Weapons:</p>
+      <% @weapons.each do |weapon|%>
+        <li>Name: <%= weapon.name %></li>
+        <li>Damage: <%= weapon.damage %></li>
+        <li>Properties: <%= weapon.properties %></li>
+        <li>Cost: <%= weapon.cost %></li>
+        <li>Weight: <%= weapon.weight %></li>
+        <li>Classification: <%= weapon.classification %></li>
+        -----------
+      <% end %>
+    <p>Armor:</p>
+      <% if @armor[0].nil? || @armor[0] == "" %>
+        <p>No Armor</p>
+      <% else %>
+        <% @armor.each do |piece|%>
+          <li>Name: <%= piece.name %></li>
+          <li>Armor Class: <%= piece.armor_class %></li>
+          <li>Cost: <%= piece.cost %></li>
+          <li>Strength Needed: <%= piece.strength %></li>
+          <li>Stealth Disadvantage: <%= piece.stealth %></li>
+          <li>Weight: <%= piece.weight %></li>
+          <li>Classification: <%= piece.classification %></li>
+          -----------
+        <% end %>
+      <% end %>
+    <p>Extras:</p>
+      <% @npc.background_equipment[2..-3].split(', ').each do |piece| %>
+        <li><%= piece %></li>
+      <% end %>
+
+      <% @pack.flatten[0].things.split(" - ").each do |thing| %>
+        <li><%= thing %></li>
+      <% end %>
+      <li><%= @npc.equipment_extras %></li>
+
+  <h3>Traits:</h3>
+    <% @npc.traits[3..-3].split(', ').each do |trait| %>
+      <p><%= trait %></p>
+    <% end %>
+
+  <h3>Proficiencies:</h3>
+    <p>Skills:</p>
+      <% @npc.stats_skills_proficiencies.split(',').each do |prof| %>
+          <li> <%= prof %></li>
+      <% end %>
+    <p>Equipment:</p>
+      <li>Armor: <%= @npc.proficiencies_armor %></li>
+      <li>Weapons: <%= @npc.proficiencies_weapons %></li>
+      <li>Tools: <%= @npc.proficiencies_tools %>, <%= @npc.background_tool_proficiency_one %>, <%= @npc.background_tool_proficiency_two %></li>
+
+  <h3>Stats:</h3>
+    <p>Hit Points: <%= @npc.stats_hit_points %></p>
+    <p>Passive Perception: <%= @npc.stats_passive_perception %></p>
+    <p>Core Stats:</p>
+      <p>Scores:</p>
+        <li>Strength: <%= @npc.stats_core_stats_stats_scores_str %></li>
+        <li>Dexterity: <%= @npc.stats_core_stats_stats_scores_dex %></li>
```

Comment:
I definitely want to shorten these names.  I just kept them like this to make it a easier to compare to the old HTML